src/overmapbuffer.cpp

```diff
@@ -661,17 +674,18 @@ std::vector<radio_tower_reference> overmapbuffer::find_all_radio_stations()
     return result;
 }
 
-city_reference overmapbuffer::closest_city( const point center )
+city_reference overmapbuffer::closest_city( const tripoint &center )
 {
     // a whole overmap (because it's in submap coordinates, OMAPX is overmap terrain coordinates)
     auto const radius = OMAPX * 2;
     // Starting with distance = INT_MAX, so the first city is already closer
-    city_reference result{ nullptr, nullptr, point( 0, 0 ), INT_MAX };
+    city_reference result{ nullptr, nullptr, tripoint( 0, 0, 0 ), INT_MAX };
     for( auto &om : get_overmaps_near( center, radius ) ) {
         const auto abs_pos_om = om_to_sm_copy( om->pos() );
         for( auto &city : om->cities ) {
             const auto rel_pos_city = omt_to_sm_copy( point( city.x, city.y ) );
-            const auto abs_pos_city = abs_pos_om + rel_pos_city;
+            const auto abs_pos_city = tripoint( abs_pos_om + rel_pos_city, 0 );
+            // TODO: Z-level cities
```

Comment:
Cities have no z-level attached, so what should be done here? The returned points could either inherit the z-level from the center-parameter, or always be 0 (as it is now). Always 0 seems the best as cities are always on the ground and the center parameter usually comes from the players location which might be at any z-level.